sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -172,17 +179,41 @@ trait WindowExecBase extends UnaryExecNode {
         // Create the factory to produce WindowFunctionFrame.
         val factory = key match {
           // Offset Frame
```

Comment:
update the comment to explain frameless

Reply:
OK. I update the comment of `FrameLessOffsetWindowFunctionFrame`